### ISSUE TITLE
TYPO3 13 Support

### ISSUE THREAD

**Issue by BokuNoMaxi** [pagemachine/typo3-formlog#233]
As noted in Slack I started to upgrade your extension to be compatible with TYPO3 13 but I am kinda stuck in replacing the frontend Javascript, so that it works again. 

The Finisher in the frontend works. so after submitting the form the record will be saved and is visible in the backend module. So only the frontend stuff is open to be implemented. 

The export button doesn't work either, because of the error in the javascript, so I couldn't test the export, but I goes there it throws an error.

Link to the fork => [https://github.com/BokuNoMaxi/typo3-formlog](url)
what I did:
- I let rector do the basic steps first and checked his changes, so there are a few code optimizations
- I updated the backendmodule of the index Action so that it works as before
- I moved the registration of the JavaScriptModules in the designated JavaScriptModules.php
- Fixed the f:be.pageRender viewhelper

**Comment by bh-teufels** [comment]
@mbrodala have you been able to look at the fork of @BokuNoMaxi what is the status here?

**Comment by mbrodala** [comment]
No, not yet. First a PR would be helpful (following the CGL) for reviewing.

**Comment by BokuNoMaxi** [comment]
hadn't had the time to dig deeper into this topic, we talked about new JavaScript libraries on slack back then, but loading them the typo3 way was a bit too much for me. In my fork everything works except the backend module.
